iotdb/src/main/java/org/apache/iotdb/db/sql/parse/AstErrorNode.java

```diff
@@ -26,7 +26,7 @@
 public class AstErrorNode extends AstNode {
 
   private static final long serialVersionUID = 1L;
-  CommonErrorNode delegate;
+  transient CommonErrorNode delegate;
```

Comment:
add comment for adding transient

Reply:
Exceptions might be thrown when serializing this class without transient.

Reply:
Added "delegate needn't be serialized."